publish-service/build.gradle

```diff
@@ -79,5 +79,12 @@ dependencies {
     compileOnly("org.springframework.boot:spring-boot-starter-tomcat")
     providedRuntime("org.springframework.boot:spring-boot-starter-tomcat")
     testCompile("org.springframework.boot:spring-boot-starter-tomcat")
-        
+
+    //gson support
+    compile('com.google.code.gson:gson:2.6.2')
```

Comment:
Can you make it consistent with the rest of the compile statement? Either we have compile("...") or compile "..."